apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.html.ts

```diff
@@ -84,6 +86,27 @@ export function InstructorQuestionPreview({
         : ''}
       <div class="row">
         <div class="col-lg-9 col-sm-12">
+          ${readmeHtml
+            ? html`
+                <div class="card mb-3">
+                  <div class="card-header bg-secondary text-white d-flex align-items-center">
+                    <button
+                      class="btn btn-secondary btn-xs show-hide-btn"
+                      data-bs-toggle="collapse"
+                      data-bs-target="#readme-card-body"
+                      aria-expanded="true"
+                      aria-controls="#readme-card-body"
+                    >
+                      <i class="fa fa-angle-up fa-fw expand-icon"></i>
```

Comment:
I'd be inclined to put this button on the right side of the card header to mirror the submission panels.